src/helpers/reader.js

```diff
@@ -59,7 +59,7 @@ function readFile(file, next) {
   fs.readFile(file, 'utf8', (err, data) => {
 
     if (err) {
-      log.error(err);
+
```

Comment:
In most of the 'helpers' we log errors. If you think that we should not, then it would be good to update the rest of the files and get rid of logging - just to be consistent across all 'helpers'